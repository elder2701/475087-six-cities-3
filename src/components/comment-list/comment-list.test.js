import React from "react";
import renderer from "react-test-renderer";
import CommentList from "./comment-list.jsx";

const comments = [
  {avatar: `img/avatar-max.jpg`,
    name: `Vaya`,
    rating: 4,
    text: `A quiet cozy and picturesque that hides behind a a river by the
  unique lightness of Amsterdam. 
  century.`,
    time: `June 3022`
  },
  {avatar: `img/avatar-max.jpg`,
    name: `Mx`,
    rating: 1,
    text: `ycentury.`,
    time: `May 2000`
  },
  {avatar: `img/avatar-max.jpg`,
    name: `Ola`,
    rating: 3,
    text: `A quiet cozy and picturesque that hides behind a a river by theYiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii.`,
    time: `April 2000`
  }
];


it(`<CommentList /> sould be render`, () => {
  const tree = renderer
      .create(<CommentList comments={comments} />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});
