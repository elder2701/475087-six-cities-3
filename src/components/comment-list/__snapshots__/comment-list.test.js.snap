// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentList /> sould be render 1`] = `
<ul
  className="reviews__list"
>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="img/avatar-max.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Vaya
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "80%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        A quiet cozy and picturesque that hides behind a a river by the
  unique lightness of Amsterdam. 
  century.
      </p>
      <time
        className="reviews__time"
        dateTime="2019-04-24"
      >
        June 3022
      </time>
    </div>
  </li>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="img/avatar-max.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Mx
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "20%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        ycentury.
      </p>
      <time
        className="reviews__time"
        dateTime="2019-04-24"
      >
        May 2000
      </time>
    </div>
  </li>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="img/avatar-max.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Ola
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "60%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        A quiet cozy and picturesque that hides behind a a river by theYiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii.
      </p>
      <time
        className="reviews__time"
        dateTime="2019-04-24"
      >
        April 2000
      </time>
    </div>
  </li>
</ul>
`;
